### PR TITLE
taxonomy: Lithuanian ingredients

### PR DESCRIPTION
**Pull request by DanieliusAsm**
### What

Added some missing ingredients for a couple of products.

### CODE REVIEW

File: taxonomies/ingredients.txt
Comment (DanieliusAsm) [inline taxonomies/ingredients.txt:1]: I added "emulsiklis lecitinai (saulėgrąžų)" under sunflower lecithin. Im not sure if the system understands that its sunflower lecithin or considers "saulėgrąžų" as sepparate ingredient

Comment (DanieliusAsm) [inline taxonomies/ingredients.txt:1]: I added "emulsiklis lecitinai (saulėgrąžų)" under sunflower lecithin. Im not sure if the system understands that its sunflower lecithin or considers "saulėgrąžų" as sepparate ingredient

Comment (DanieliusAsm) [inline taxonomies/ingredients.txt:1]: how do you define taxonomy ingredients when the ingredient is listed as: "starch (potato, rice)" in lithuanian "krakmolas (bulvių, ryžių)" ? will it look for "potato starch" and "rice starch" ?  also "stabilizatoriai (e407a, e450)" I guess the system should look for "stabilizatoriai e407a" and "stabilizatoriai e450" ?

**Review by stephanegigandet — APPROVED** [review]
Thank you!

**Pull request merged**